README.md

```diff
@@ -27,52 +27,52 @@ struct PlayerResolver: Resolver {
 }
 ```
 
-This is a trivial example which on its own may not warrant a special "Resolver," but as we'll see below in the "Who is Settler For?" section, a Resolver can be particularly helpful in the development and maintenance of _large_, _complex_, and _interconnected_ dependency graphs.
+This is a trivial example which on its own may not warrant a special “resolver,” but as we’ll see below in the “Who is Settler for?” section, a resolver can be helpful in the development and maintenance of large and dense dependency graphs.
 
-See [Resolver.md](https://github.com/daltonclaybrook/Settler/blob/main/Resolver.md) for an in-depth Resolver implementation guide.
+See [Resolver.md](https://github.com/daltonclaybrook/Settler/blob/main/Resolver.md) for an in-depth `Resolver` implementation guide.
 
-See the [SettlerDemo directory](https://github.com/daltonclaybrook/Settler/tree/main/Sources/SettlerDemo) for a more detailed example of a Resolver.
+See the [SettlerDemo directory](https://github.com/daltonclaybrook/Settler/tree/main/Sources/SettlerDemo) for a more detailed resolver example.
 
 ## Compiler magic
 
-The power (and magic ✨) of Settler lies in its ability to parse your Resolvers in-line with the Swift compiler and report errors directly in Xcode as if it were part of the compiler itself.
+The power (and magic ✨) of Settler lies in its ability to parse your resolvers alongside the Swift compiler and report errors directly in Xcode as if it were part of the toolchain itself.
 
-By defining each of your dependencies as a function, and by using `Key` members as inputs/outputs of those functions, Settler is able to resolve your entire dependency graph in the correct order, ignoring types that are unused, lazily initializing dependencies where necessary, and reporting errors when things are wrong or missing.
+By defining your dependencies as a function, and by using `Key` members as inputs and outputs of those functions, Settler is able to resolve your dependency graph in the correct order, ignoring types that are unused, lazily initializing dependencies where necessary, and reporting errors when things aren’t quite right.
 
-Once configured as a Run Script build phase in Xcode, Settler can report errors in your `Output` and `Key` types, whether your resolver functions contain invalid parameters, whether any dependencies are missing a corresponding function (or are duplicated), whether you have a circular dependency, and many more. Rest assured that if you see no reported errors in Xcode, your Resolver is correct.
+Once configured as a Run Script build phase in Xcode, Settler can report errors in your `Output` and `Key` types, whether your resolver functions contain invalid parameters, whether any dependencies are missing their corresponding function (or are duplicated), whether there’s a circular dependency, and much more. Rest assured that if you see no reported errors, your `Resolver` implrementation is correct.
```

Comment:
>implrementation

As a Texas, I probably do put a lot of "r"s in things that don't need them. Not sure if it works here, though. 😆